.github/dependabot.yml

```diff
@@ -0,0 +1,35 @@
+# Please see the documentation for Dependabot configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: /
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+    labels:
+      - "auto-approve"
+      - ":robot: bot"
+    open-pull-requests-limit: 5
+    # Include a list of updated dependencies
+    # with a prefix determined by the dependency group
+    commit-message:
+      prefix: "chore(deps):"
+      include: "scope"
+    target-branch: "development"
+  - package-ecosystem: "github-actions"
+    directory: "/"
```

Comment:
Looks like the directory needs to be `.github/workflows` for github actions according to this comment https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#directory